### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,43 +2,40 @@
...
-When set to true, the dropbox app(s) won't show a notification for this upload
+The Dropbox app won't show a notification for this upload.
 
 Used to avoid this:
 
 ![double](media/double.png)
-
-## License
-
-MIT © [George Karagkiaouris](https://gkaragkiaouris.com)
```
Comment (sindresorhus) [inline readme.md:44]: Removed it as it moot now that GitHub shows the license in the UI itself. I'm removing it from all my readme's too. Let me know if you still want to keep it though.

File: readme.md [hunk 2]
```diff
@@ -2,43 +2,40 @@
...
 Default: `true`
 
-When set to true, if there is a conflict (the file already exists) the new upload will automatically be renamed in order to keep both
+If there is a conflict (the file already exists), the new upload will automatically be renamed in order to keep both.
 
 #### mute
-Type: `Boolean`
+
+Type: `boolean`\
 Default: `false`
```
Comment (sindresorhus) [inline readme.md:35]: This should really be `true` by default.

File: readme.md [hunk 3]
```diff
@@ -2,43 +2,40 @@
...
 
 #### path
-Type: `String`
 
-When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called Kaptures, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
+Type: `string`
+
+When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called `Kaptures`, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
 
 #### autorename
```
Comment (sindresorhus) [inline readme.md:25]: Can you change this to `autoRename`? It bothers me deeply :P

File: readme.md [hunk 4]
```diff
@@ -2,43 +2,40 @@
...
-Available options:
+Alternatively, you can manually edit the options in your text editor; Click `Open plugins folder` and edit the `kap-dropbox.json` file to change options.
 
 #### path
-Type: `String`
 
-When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called Kaptures, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
+Type: `string`
+
+When provided, uploads will be placed in that directory. For example, to upload recordings under a directory called `Kaptures`, set `path` to `/Kaptures`. If the directory doesn't exist it will be created.
```
Comment (sindresorhus) [inline readme.md:23]: Does it really have to be `/Kaptures` with a `/` prefix?

File: readme.md [hunk 5]
```diff
@@ -2,43 +2,40 @@
...
 
 In the editor, after recording, select a format, and then `Upload to Dropbox`.
 
 ## Options
 
-In the `plugins` pane, click `Open plugins folder` and edit the `kap-dropbox.json` file to change options
+In the “Plugins” pane, click the pencil icon to edit the plugin options.
 
-Available options:
+Alternatively, you can manually edit the options in your text editor; Click `Open plugins folder` and edit the `kap-dropbox.json` file to change options.
```
Comment (sindresorhus) [inline readme.md:17]: I just noticed that we no longer have this button, so maybe the whole section here should be removed?

Comment (karaggeorge) [reply readme.md:17]: We moved it into each plugin's config window instead. I'll update 👍

File: readme.md (hunk 4 above)
Comment (karaggeorge) [reply readme.md:23]: It's the way the API expects it, so I kept it the same

File: readme.md (hunk 5 above)
Comment (sindresorhus) [reply readme.md:17]: Ah, right. Duuh.

**Pull request merged**